cloudmarker/alerts/emailalert.py

```diff
@@ -15,7 +15,7 @@ class EmailAlert:
     """A plugin to send email notification for anomalies found."""
 
     def __init__(self, host, port, subject, to, sender, body, use_ssl=True,
-                 username=None, password=None):
+                 username=None, password=None, debug=False):
```

Comment:
I was hoping to keep it internal like `_debug` the way it is for `azcloud` plugin. We can decide based on @susam 's opinion. Since this would only be meant for internal development and testing and not for prod use.

Reply:
I like `debug` because this can be used by the user too for debugging purpose.

By the way, this pull request would be overwritten by #108.